ouroboros-consensus-diffusion/src/ouroboros-consensus-diffusion/Ouroboros/Consensus/Node.hs

```diff
@@ -284,6 +289,21 @@ data LowLevelRunNodeArgs m addrNTN addrNTC versionDataNTN versionDataNTC blk
     , llrnPublicPeerSelectionStateVar :: StrictSTM.StrictTVar m (Diffusion.PublicPeerSelectionState addrNTN)
     }
 
+data NodeDatabasePaths =
+    OnePathForAllDbs FilePath
+  | MultipleDbPaths
+      FilePath -- ^ Immutable path, usually pointing to a non-necessarily
+               -- performant volume
+      FilePath -- ^ Volatile path, usually pointing to a performant volume
```

Comment:
I think it makes to make it clear that this is not just for the VolatileDB, but for everything apart from the ImmutableDB. Maybe even call it "non-immutable"? Or go even further and have a record type with different `FilePath`s for each component?

Reply:
I called it `nonImmutablePath`, I think it is enough this way.